Snowflake.Data/Core/Session/SFSession.cs

```diff
@@ -77,6 +85,10 @@ internal void ProcessLoginResponse(LoginResponse authnResponse)
                 serverVersion = authnResponse.data.serverVersion;
                 masterValidityInSeconds = authnResponse.data.masterValidityInSeconds;
                 UpdateSessionParameterMap(authnResponse.data.nameValueParameter);
+                if (_disableQueryContextCache)
+                {
+                    logger.Debug("Query context cache disabled.");

```

Comment:
We only log here and do nothing more?

Reply:
Yes. Other query context functions (like UpdateQueryContextCache and GetQueryContextRequest) will check the flag and do nothing when it's turned on, but we only need to log that once per connection, instead of each time check the flag per request.